src/api/spec/components/previews/status_message_component_preview.rb

```diff
@@ -0,0 +1,14 @@
+class StatusMessageComponentPreview < ViewComponent::Preview
```

Comment:
Do you think that view component previews are useful? We could eventually replace https://obs-patterns.netlify.app/ with those previews.